app/controllers/resumes_controller.rb

```diff
@@ -11,8 +11,10 @@ def index
   end
 
   def show
-    # @resume = Resume.create(target_job: "Rockstar", document_data: "Who runs the world? Squirrels!")
-    # render json: @resume
+    @resume = Resume.find(params[:id])
```

Comment:
There is a before_action above the controller that helps find the '@resume' already.